src/components/Button/Button.js

```diff
@@ -53,20 +54,19 @@ class Button extends React.PureComponent {
     const {
       children,
       text,
-      type,
       primary,
-      disabled,
       danger,
       selected,
-      style,
-      className,
+      disabled,
+      ...otherProps,
     } = this.props;
 
+    const buttonProps = omit(otherProps, ['onClick']);
```

Comment:
Could we also omit the `style` prop? :) It's not cool that it's overridable (and also couldn't find any place in the code where it's actually being overriden).

Reply:
> couldn't find any place in the code where it's actually being overriden

we have the best coders!

I agree we shouldn't use it... unless its an emergency ofc. I guess because we accept className you can styled(Button) it...